test_fms/parser/test_yaml_parser.F90

```diff
@@ -115,7 +115,7 @@ program test_read_and_parse_file
 
 !! Try get_value_from_key using a r8 buffer
 call get_value_from_key(yaml_file_id1, variable_ids(1), "fill_value", r8_buffer)
-if (r8_buffer .ne. real(-999.9, kind=r8_kind)) call mpp_error(FATAL, "fill_value was not read correctly as an r8!")
+if (abs(r8_buffer - real(-999.9, kind=r8_kind)) .gt. 5e-5) call mpp_error(FATAL, "fill_value was not read correctly as an r8!")
```

Comment:
Since this is using `real*8`s, I think you want to change the `e` to a `d`
```fortran
abs(r8_buffer - real(-999.9, kind=r8_kind)) .gt. 5d-5
```

Reply:
@rem1776 This will need to be done in main first.

Reply:
@uramirez8707 @thomas-robinson Just put in #886

Reply:
@rem1776 #886 is approved. 
@uramirez8707 are you going to merge it in and then we can merge this PR?